Gruntfile.js

```diff
@@ -14,7 +14,9 @@ const extract = require('jsxgettext-recursive');
 // use this if you want to recursively match all subfolders:
 // 'test/spec/**/*.js'
 module.exports = function (grunt) {
-  'use strict';
+  'use strict'
```

Comment:
+1

@TDA: Let's remove the semicolons from the Gruntfile.js and tasks/eslint.js and remove that `semi` rule from the .eslintrc file.

![](http://jvtrigueros.github.io/dcc4-function-composition-scala/images/no-semicolon.jpg)